ansible/roles/apache/defaults/main.yml

```diff
@@ -860,6 +893,28 @@ apache__vhost_allow_override: 'None'
 # Refer to the `Apache Options directive documentation`_ for details.
 apache__vhost_options: [ '+FollowSymLinks' ]
                                                                    # ]]]
+                                                                   # ]]]
+# .. envvar:: apache_webroot_create [[[
+#
+# Create global webroot directories?
+# Path: :file:`/srv/www/sites/*/public`.
+apache_webroot_create: True
+
+                                                                   # ]]]
+# .. envvar:: apache_webroot_owner [[[
+#
+apache_webroot_owner: 'root'
+
+                                                                   # ]]]
+# .. envvar:: apache_webroot_group [[[
+#
+apache_webroot_group: 'root'
+
+                                                                   # ]]]
+# .. envvar:: apache_webroot_mode [[[
+#
+apache_webroot_mode: '0755'
```

Comment:
Those variables should start with `apache__`.

Reply:
That should be resolved in the latest commit.